src/app/onion/core/learning-object.service.ts

```diff
@@ -143,26 +138,6 @@ export class LearningObjectService {
         return response;
       });
   }
-  /**
-   * Fetches Learning Object by ID (full)
-   *
-   * @param {string} learningObjectID
-   * @returns {Promise<LearningObject>}
-   * @memberof LearningObjectService
-   */
-  getLearningObject(learningObjectId: string): Promise<LearningObject> {
```

Comment:
Was this not being used?

Reply:
mentioned this in the standup: this route was used when we used to fetch learning objects by their IDs, not their CUIDs. so the logic that use this function and the function itself were removed as the need for querying by ID isn't really needed anymore.